src/ceph-disk/ceph_disk/main.py

```diff
@@ -3334,14 +3334,21 @@ def main_activate(args):
         else:
             raise Error('%s is not a directory or block device' % args.path)
 
+        # exit with 0 if the journal device is not up, yet
+        # journal device will do the activation
+        osd_journal = '{path}/journal'.format(path=osd_data)
```

Comment:
To be fair, it is not a regression in this respect either -- ceph-disk have passed the --osd-journal argument with this path even before this (old line 3344). If the file does not exist, the next if will simply fail (it is not a link if it does not exist) and that osd_journal path will be sent to ceph-osd --osd-journal flag just like before.